### DISCUSSION TITLE
Possible to fixate layout?

### DISCUSSION THREAD

**Discussion by wouterio** [foxglove/community#17] [Q&A] [answered]
Hi,

We've been using Foxglove Studio for a bit now, and we're quite happy about it. So now we want to find out if we can build a customer GUI based on Foxglove Studios through which they can interact with our robot. We like that Foxglove Studios' layout is very modular and flexible. But to keep things simple for our customers, we need to fixate the layout and leave out all options to change it.

So our questions are:
- Is there already a way to fixate the layout? Something like 'Save as fixed layout'?
- If not, would it be a nice feature to fixate a certain layout?
- Is it enough if the buttons to add/remove/configure panels can be disabled? And to do so, what would be the right place to start?

If you have questions, please let me know.
Kind regards.

**Comment by jtbandes** [comment, marked as answer]
Disabling editing is not possible today. See: 
- https://github.com/foxglove/studio/issues/1071

For sharing layouts between team members, see [Team layouts](https://foxglove.dev/docs/studio/layouts#team-layouts)

**Comment by wouterio** [comment]
Ok thanks for clarifying.
